engines/director/images.cpp

```diff
@@ -319,4 +319,52 @@ bool BITDDecoder::loadStream(Common::SeekableReadStream &stream) {
 	return true;
 }
 
+void copyStretchImg(Graphics::Surface *srcSurface, Graphics::Surface *targetSurface, const Common::Rect &srcRect, const Common::Rect &targetRect, const byte *pal) {
+	if (!(srcSurface) || !(targetSurface))
+		return;
+
+	if (targetRect.width() != srcRect.width() || targetRect.height() != srcRect.height()) {
```

Comment:
Why not use `Surface::scale()`?

Reply:
Hah, I spent a bit of time scraping through the project looking for a scaling function (because why wouldn't we have one??) but failed. Serves me right for looking for the word "stretch" instead of "scale".
The output looks to be the same as real D4, so let's use that.